assets/src/containers/ResourcesAccessed.js

```diff
@@ -213,21 +217,29 @@ function ResourcesAccessed (props) {
     }
   }
 
-  const ResourceAccessChartBuilder = (resourceData) => {
+  const resourceAccessChartBuilder = (resourceData, resourcesLimit) => {
     if (resourceTypeFilter.length === 0) {
       return (<AlertBanner>Please select a resource type to display data.</AlertBanner>)
     } else if (!resourceData || Object.keys(resourceData).length === 0) {
       return (<AlertBanner>Resource data for your selections is not available.</AlertBanner>)
     } else {
       return (
-        <ResourceAccessChart
-          data={resourceData}
-          weekRange={weekRange}
-          gradeSelection={resourceGradeFilter}
-          resourceType={resourceTypeFilter}
-          aspectRatio={0.3}
-          minHeight={350}
-        />
+        <>
+          <div style={{ textAlign: 'center', verticalAlign: 'middle' }}>
+            {typeof resourcesLimit === 'undefined' ? <div>Displaying all available resources in this course</div> :
+              <div>Displaying top {resourcesLimit} most accessed resources in this course</div>
```

Comment:
I think text should go in a different tag than `div`. You could also make use of Material UI's `Typography`.

Reply:
![Screenshot 2023-03-09 at 11 24 33 AM](https://user-images.githubusercontent.com/35741256/224092812-ccc9f277-5da7-4b2b-9aab-9b0e5097a14f.png)

Looks okay. I'd consider adding a little margin on the bottom after the text. This may be handled for you if you use `Typography`, or you may have more utilities available to you.

Reply:
I can ask at the meeting!

Reply:
Jennifer said it's better to leave both in there so there's no confusion when it doesn't appear. Marking as resolved.